### DISCUSSION TITLE
Horizontal scroll on Screen/Layout Sidebar

### DISCUSSION THREAD

**Discussion by hardc0l2e** [Budibase/budibase#2234] [Ideas]
Screen with  a little bit of content is  a bit hard to manage, is it possible to have a horizontal scroll on that section, no need to resize the frame of the builder, just a horizontal screen will do.

![image](https://user-images.githubusercontent.com/1472421/128009760-b9a6deb3-d141-45d4-a3ca-13e285bad90b.png)

**Comment by hardc0l2e** [comment]
When I adjust the CSS in browser inspect, I can add the horizontal scroll "width : fit-content"

![20210803_07 53 55 PM-Budibase_-_Google_Chrome](https://user-images.githubusercontent.com/1472421/128010937-822db18e-7e20-4c97-bbd9-8686742980f3.png)
![20210803_07 54 05 PM-Budibase_-_Google_Chrome](https://user-images.githubusercontent.com/1472421/128010932-22533bed-b98d-4869-8197-de4d72b84407.png)

**Comment by shogunpurple** [comment]
Hi @hardc0l2e - we have an issue for this one [here](https://github.com/Budibase/budibase/issues/2159) if you would like to track!
